styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,15 @@
-.footer{
-  
-}
\ No newline at end of file
+.rug{
```

Comment:
This should be in `rug.scss` as there is a lot of work going into this file in #109